docs/usage.rst

```diff
@@ -5,20 +5,73 @@ Usage
 Authentication
 --------------
 
-To access an Artifactory server,
-store your username and `API key`_ in :file:`~/.artifactory_python.cfg`
+Artifactory servers can allow anonymous access
+by logging in with a fixed pair
+of ``'anonymous'`` as username
+and ``''`` as password.
+This is the default behavior of :mod:`audfactory`.
+
+.. jupyter-execute::
+
+    import audfactory
+
+    audfactory.authentification('https://artifactory.domain.com/artifactory')
+
+To access an Artifactory server
+that requires logging in with a username and password,
+store your username and `API key`_
+in :file:`~/.artifactory_python.cfg`
+using separate sections for every server.
+Every section is marked
+by the server URL in square brackets
+without the ``https://`` or ``http://``
+at the beginning:
 
 .. code-block:: cfg
 
-    [artifactory.audeering.com/artifactory]
-    username = MY_USERNAME
-    password = MY_API_KEY
+    [artifactory1.domain.com/artifactory]
+    username = MY_USERNAME1
+    password = MY_API_KEY1
+
+    [artifactory2.domain.com/artifactory]
+    username = MY_USERNAME2
+    password = MY_API_KEY2
+
+.. jupyter-execute::
+    :hide-code:
+    :hide-output:
+
+    import os
+
+    os.environ['ARTIFACTORY_USERNAME'] = 'MY_USERNAME2'
```

Comment:
Setting these env variables is not required in case you have `~/.artifactory_python.cfg` configured. These are alternatives.

Reply:
As you realized in your comment below, these settings are only workarounds to make the code execution in the documentation show the right output without having an actual config file.
I added comments to the sections to highlight that more.

The alternative would be to generate the config file on the fly here. Would you prefer this for better understanding?